src/tools/debugger/session/Breakpoints.java

```diff
@@ -54,6 +54,7 @@
 
   /** Manually managed by us, instead of Truffle. */
   private final Map<FullSourceCoordinate, BreakpointEnabling<SectionBreakpoint>> channelOppositeBreakpoints;
+  private final Map<FullSourceCoordinate, BreakpointEnabling<SectionBreakpoint>> onExecutionBreakpoints;
```

Comment:
It's for the breakpoints triggered when an activity starts executing, the `ACTIVITY_ON_EXEC` breakpoints/steps.
So, it's for processes, tasks, and threads.